packages/components/src/components/post-popovercontainer/post-popovercontainer.tsx

```diff
@@ -117,7 +123,7 @@ export class PostPopovercontainer {
   /**
    * Animation style
    */
-  @Prop() readonly animation?: 'pop-in' | null = null;
+  @Prop() readonly animation?: AnimationName | 'none' | null = null;
```

Comment:
We already have `null` as the value if no animation should be applied, why do we need to add `none`? This seems like an unwanted workaround for the validation.